acstools/polarization_tools.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""
+Toolkit for analyzing HST/ACS polarization data.
+
+CHANGE LOG
+----------
+2020-09-22: Creation. (T. Desjardins)
+"""
+
+import numpy as np
+import os
+import yaml
+
+from astropy.table import Table
+
+
+class PolarizerTables:
+    """
+    A simple class for holding all of the polarization tables (as astropy tables) in attributes.
+    """
+    def __init__(self):
+        with open(os.path.normpath(os.path.join(os.path.dirname(__file__), os.pardir, 'data',
+                                                'polarizer_tables.yaml')), 'r') as yf:
+            self.table_data = yaml.safe_load(yf)
+
+        self.wfc_transmission = Table(self.table_data['transmission']['wfc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'))
+
+        self.hrc_transmission = Table(self.table_data['transmission']['hrc'],
+                                      names=('filter', 't_para', 't_perp', 'correction'))
+
+        self.wfc_efficiency = Table(self.table_data['efficiency']['wfc'],
+                                    names=('filter', 'pol0', 'pol60', 'pol120'))
+
+        self.hrc_efficiency = Table(self.table_data['efficiency']['hrc'],
+                                    names=('filter', 'pol0', 'pol60', 'pol120'))
+
+
+class Polarization:
+    """
+    Class for handling ACS polarization data.
+
+    Examples
+    --------
+
+    >>> from acstools.polarization_tools import Polarization
+    >>> vela_181 = Polarization(63684, 67420, 63752, 'F606W', 'WFC', 348.084)
+    >>> vela_181.calc_stokes()
+    >>> vela_181.calc_polarization()
+    >>> print(f'I = {vela_181.stokes_i:.2f}, Q = {vela_181.stokes_q:.2f}, U = {vela_181.stokes_u:.2f}')
+    >>> print(f'Polarization: {vela_181.polarization * 100:.2f} %, Angle: {vela_181.angle:.2f} deg')
+    """
+
+    def __init__(self, pol0, pol60, pol120, filter_name, detector, pav3):
+        """
+        Parameters
+        ----------
+        pol0 : float
+            Photometric measurement in POL0 filter. Units: electrons or electrons/second.
```

Comment:
Why not use `astropy.units.Quantity`? Same question for the other parameters with assumed units.

Reply:
Input values can be counts or rate, as long as they're consistent. And the values they transform into do not have units that transform logically. For example, Stokes I is unitless, but is the sum of three values that are either in counts or rate. So I think it's better to not accept `astropy.units.Quantity` objects for now, but I do think it's good to output the position angle as a quantity object to be clear it is in degrees.